src/field_theory/adjoin.lean

```diff
@@ -757,6 +757,37 @@ end
 
 end alg_hom_mk_adjoin_splits
 
+section finite_dimensional_sup
+
+lemma le_sup_to_subalgebra {K L : Type*} [field K] [field L] [algebra K L]
```

Comment:
This doesn't really make sense in this section, as it's not about finite dimensionality